src/dune_rules/coq_rules.ml

```diff
@@ -86,17 +88,18 @@ module Bootstrap = struct
       | true -> Bootstrap_prelude)
 
   let boot_lib_flags ~coqdoc lib =
-    let open Command in
     let dir = Coq_lib.src_root lib in
-    if coqdoc then [ Args.A "--coqlib"; Args.Path (Path.build dir) ]
-    else Args.A "-boot" :: Util.theory_coqc_flag lib
+    let open Command.Args in
```

Comment:
We prefer to annotate the return type or qualify the constructors instead of `let open`.

Reply:
Done